components/centraldashboard/public/components/main-page.pug

```diff
@@ -17,8 +17,10 @@ app-drawer-layout.flex(narrow='{{narrowMode}}',
                     paper-item.menu-item [[item.text]]
         aside.flex
         footer.footer
-            section.privacy
-                a(title='Kubeflow Privacy Policy', href='https://policies.google.com/privacy') Privacy
+            section.information
+                a.privacy(title='Kubeflow Privacy Policy', target='_blank', href='https://policies.google.com/privacy') Privacy
+                .bullet
+                a.usage(title='Kubeflow Usage Reporting', target='_blank', href='https://master.kubeflow.org/docs/other-guides/usage-reporting/') Usage Reporting
```

Comment:
Should this explicitly be master.kubeflow? The **www** subdomain (https://www.kubeflow.org/docs/other-guides/usage-reporting/) seems to work as well.

Reply:
I ran a search, and got back master. But this is the preferred link, and I've submitted the commit. I just confirmed with @rileyjbauer.

Reply:
Which is preferred, and preferred by whom? It looks like this PR was still using master.

Reply:
Yeah that was a mistake. I misinterpreted what AP had said as meaning that the current commit with (master) was the correct link. He said he will making a subsequent PR.